src/types/queryRequestDBParams.ts

```diff
@@ -14,8 +14,7 @@ export const isQueryRequestDBParams = (
     typeof test?.recipientName === 'string' &&
     typeof test?.requesterEmail === 'string' &&
     typeof test?.requesterName === 'string' &&
-    typeof test?.dateFrom === 'string' &&
-    typeof test?.dateTo === 'string' &&
+    // TODO: add a check for the "dates" property
```

Comment:
Another question about whether we need to do this in this PR?

Reply:
Actually, I realised that this type isn't used at all now, so I've removed it! I have though done the check I was talking about in the real place where we're checking types after retrieving from the database